.github/workflows/jni-docker-build.yml

```diff
@@ -0,0 +1,40 @@
+name: JNI Docker Build
```

Comment:
Missing copyrights

Reply:
I was trying to follow existing cudf workflow files https://github.com/rapidsai/cudf/tree/branch-22.06/.github/workflows
not sure if we need to add the license header here

Reply:
added